k8s/helm/hypercode/templates/healer-deployment.yaml

```
@@ -0,0 +1,51 @@
+    matchLabels:
+      {{- include "hypercode.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: healer
+  strategy:
+    type: Recreate  # Healer must always be running — never zero replicas
+  template:
+    metadata:
+      labels:
+        {{- include "hypercode.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: healer
+    spec:
+      serviceAccountName: {{ include "hypercode.serviceAccountName" . }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)